Exec/science/Detonation/inputs-collision

```diff
@@ -53,6 +51,11 @@ amr.refine.dengrad_rel.max_level = 0
 amr.refine.dengrad_rel.relative_gradient = 0.5
 amr.refine.dengrad_rel.field_name = density
 
+amr.refine.dxnucerr_rel.max_level = 20
```

Comment:
This has a typo, but you can also just delete this line.

Reply:
fixed